features/exploredash/src/main/java/org/dash/wallet/features/exploredash/repository/AssetExploreRepository.kt

```diff
@@ -34,7 +34,11 @@ private val log = LoggerFactory.getLogger(AssetExploreDatabase::class.java)
 
 class AssetExploreDatabase @Inject constructor(@ApplicationContext context: Context) : ExploreRepository {
 
-    var contextRef: WeakReference<Context> = WeakReference(context)
+    private var contextRef: WeakReference<Context> = WeakReference(context)
+
+    private val moshi: Moshi = Moshi.Builder().addLast(KotlinJsonAdapterFactory()).build()
```

Comment:
Making Moshi instance class level field speed-up the sync process of Dash Direct data by around 30%